xarray/tests/test_datatree.py

```diff
@@ -0,0 +1,30 @@
+import xarray.testing as xrt
+from xarray import Dataset
+from xarray.tests import requires_datatree
+
```

Comment:
since the whole module depends on `datatree`, I'd call `pytest.importorskip("datatree")` somewhere at the top of the module:
```suggestion
pytest.importorskip("datatree")

```

then we don't need to decorate *every* test with `requires_datatree`

If we want to reuse `requires_datatree`, we can use:
```suggestion
pytestmark = [requires_datatree]

```